### PR TITLE
Reserved peers, reserved-only flag

### PR DESCRIPTION
**Pull request by rphmeier**
No RPC API just yet, that will come in a follow-up.

### CODE REVIEW

File: parity/cli.rs [hunk 1]
```diff
@@ -236,6 +240,8 @@ pub struct Args {
 	pub flag_no_discovery: bool,
 	pub flag_nat: String,
 	pub flag_node_key: Option<String>,
+  pub flag_reserved_peers: Option<String>,
```
Comment (tomusdrw) [inline parity/cli.rs]: Indentation?

**Comment by rphmeier** [comment]
the indentation commit was [ci skip]. The other commits still have a travis build pending. don't merge until it is finished

**Comment by rphmeier** [comment]
This needs reverting -- the travis build never finished

**Comment by debris** [comment]
ahh.. haven't noticed that. sry

**Comment by rphmeier** [comment]
superseded by #1350

**Pull request merged**